bundles/automation/org.eclipse.smarthome.automation.rest/src/main/java/org/eclipse/smarthome/automation/rest/internal/RuleResource.java

```diff
@@ -1,4 +1,8 @@
 /**
+ * Copyright (c) 2017 by Bosch Software Innovations GmbH. All rights reserved,
```

Comment:
@danchom If I can help in any way with the clarification, please let me know. And remember that we soon will have different headers anyhow...